dll/win32/shell32/lang/en-US.rc

```diff
@@ -88,6 +88,16 @@ BEGIN
     END
 END
 
+IDM_RECENTMENUHEAD MENU
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM "My Documents", IDM_MYDOCUMENTS
+        MENUITEM "My Pictures", IDM_MYPICTURES
```

Comment:
@katahiromz is it possible to generate menu dynamically by reusing existing translated strings to avoid adding duplicated strings?
- https://github.com/katahiromz/reactos/blob/32d5b74cec35bdd1fbe073e693b201a110b222e9/dll/win32/shell32/lang/en-US.rc#L833
- https://github.com/katahiromz/reactos/blob/32d5b74cec35bdd1fbe073e693b201a110b222e9/dll/win32/shell32/lang/en-US.rc#L851

Reply:
Possible, but different from Win2k3.

Reply:
Possible, but different from Win2k3.

Reply:
Created #2705.